### DISCUSSION TITLE
group fflib-apex-extensions and samplecode under a dedicated GitHub Organization

### DISCUSSION THREAD

**Discussion by PeterLin888** [wimvelzeboer/fflib-apex-extensions#11] [Q&A] [answered]
Hi William,
Would you consider group fflib-apex-extensions and samplecode repositories under a dedicated GitHub Organization before merging them into fflib-apex-common?
Thanks,
Peter

**Comment by wimvelzeboer** [comment, marked as answer]
Yes, my preference would be to move the [fflib-apex-extensions](https://github.com/wimvelzeboer/fflib-apex-extensions) and [fflib-apex-extension-samplecode](https://github.com/wimvelzeboer/fflib-apex-extensions-samplecode) to the [apex-enterprise-patterns](https://github.com/apex-enterprise-patterns) GitHub Organization.

My hope that these two repositories will get an active community, and that can then be moved.
Creating a separate GitHub Organization is I think a this stage less preferable.

**Reply by PeterLin888** [reply]
Thank you, William!
